uqcsbot/scripts/news.py

```diff
@@ -0,0 +1,39 @@
+from uqcsbot import bot, Command
+from uqcsbot.utils.command_utils import loading_status
+from typing import Tuple, Dict, List
+
+import feedparser
```

Comment:
Does this need to be added as a dependency somewhere? (I think yes, but the tests seem to be passing 🤔)

Reply:
`feedparser` is already listed explicitly in `setup.py`, so no worries there.